### ISSUE TITLE
Prompt hard to read because it uses hardcoded colors

### ISSUE THREAD

**Issue by xi** [ipython/ipython#12442]
I find some colors are hard to read, especially the red used for the output when using a dark background. The [contrast ratio is only 2.35](https://contrast-ratio.com/#%23990000-on-black) which is too low according to [WCAG21](https://www.w3.org/TR/WCAG21/#contrast-minimum).

![barely readable red output ptompt on black background](https://user-images.githubusercontent.com/202576/87507527-d4137000-c66d-11ea-8293-8e3b2b6f406c.png)

A similar issue was discussed a while ago in #9670. However, I feel like a crucial aspect was missed: It is impossible to guarantee sufficient contrast if you hardcode colors and don't know the background. So IMHO the better fix would be to not hardcode any colors and use ANSI colors instead.
